cockpit/devices/boulderSLM.py

```diff
@@ -44,12 +44,45 @@
 
 from cockpit import events
 import cockpit.gui.device
-import cockpit.gui.guiUtils
-import cockpit.gui.toggleButton
 import cockpit.handlers.executor
 import time
 import cockpit.util
 
+class LastParameters(object):
```

Comment:
Making it private might make the circular reference thing (commented above) a non-issue, too?

Reply:
No. The technical issue of circular references here is that it prevents garbage collection. That can be solved by using `weakref'. But circular references also point to very tighly coupling of modules which in this case makes sense because they this class is only meant to be used by instances of BoulderSLM.

Reply:
Fixed in 5fdb2dd991c03670fe30e662fadbab4200384f9c.